plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageRecorder.java

```diff
@@ -63,6 +63,9 @@
  */
 @SuppressWarnings("checkstyle:ClassFanOutComplexity")
 public class CoverageRecorder extends Recorder {
+    // TODO: make customizable?
```

Comment:
Yes, I think someone else added a PR to make that configurable in the existing publisher.